llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -435,6 +435,50 @@ def IREELinalgExt_TopkOp : IREELinalgExt_Op<"topk",[
   }];
 }
 
+def IREELinalgExt_PackOp : IREELinalgExt_Op<"pack",[
+  DeclareOpInterfaceMethods<LinalgExtInterface>,
+  DeclareOpInterfaceMethods<TilingInterface,
+    ["getIterationDomain",
+     "getLoopIteratorTypes",
+     "generateScalarImplementation"]>,
+  DeclareOpInterfaceMethods<MemoryEffectsOpInterface>, 
+  SingleBlockImplicitTerminator<"::mlir::iree_compiler::IREE::LinalgExt::YieldOp">,
+]>{
+  let summary = "relayout - pack: tensor expand + tensor transpose";
+  
+  let arguments = (ins Variadic<AnyShaped>:$inputs,
```

Comment:
Do we want to support multiple inputs/outputs? What's even the semantics in that case? Do all tensors need to be the same shape, or just compatible pair-wise?

Reply:
I would limit to 1 input / 1 output.

Reply:
me too